### PR TITLE
Adds Steppy the alligator and Atlas the tegu.

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About the Pull Request

Adds alligator and tegu mobs into the game, including unique Steppy and Atlas from the old repository.

## Why It's Good For The Game

Literally an alligator. Who wouldn't want a pet alligator or tegu?????

image
image

## Did you test it?

It is tested well enough. Mobs are spawnable, they move, don't cause runtimes and look nice.

## Changelog

:cl:
rscadd: Added Atlas the tegu, and Steppy the alligator.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/retaliate/tegu.dm [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	health = 80
+	maxHealth = 80
+	speed = 3
+	pry_time = 7 SECONDS
+	pry_desc = "biting"
+	ai_holder_type = /datum/ai_holder/simple_animal/retaliate
+
+/mob/living/simple_animal/hostile/retaliate/tegu/atlas
+	name = "Atlas"
+	desc = "That's the captain's small, but mighty pet tegu. They may or may not have the blood of greytiders on them."
```
Comment (Cupax3) [inline code/modules/mob/living/simple_animal/hostile/retaliate/tegu.dm]: Greytiders aren't a thing on Bay, please change.

**Review by Cupax3 — CHANGES_REQUESTED** [review]
Squash please.

**Review by Cupax3 — APPROVED** [review]
Approved, do not merge until squashed.

**Comment by EgorDinamit** [comment]
Oh for fucks sake

**Comment by EgorDinamit** [comment]
Man, should've never tried squashing commits.

**Pull request closed without merging**